README.md

```diff
@@ -14,6 +14,8 @@ The default behaviour is to use the [latest crystal release](https://github.com/
 
 To learn more about using custom buildpacks in Heroku, read [their docs](https://devcenter.heroku.com/articles/third-party-buildpacks#using-a-custom-buildpack).
 
+See also this blog post: http://crystal-lang.org/2016/05/26/heroku-buildpack.html
```

Comment:
Sure, I get your point and I understand it becomes a link, but I much rather have the links on words, tbh. Also, the repo is now in the crystal-lang organization on github which should give you some degree of confidence about it, furthermore some browsers like chrome show the url of a link on hover.

This obviously an opinion, but I think the visual nicety of a link on a word usually outweighs the added "confidence" of seeing the url.

But again, thanks for the contribution.